distmv/normal.go

```diff
@@ -75,11 +75,17 @@ func NewNormal(mu []float64, sigma mat64.Symmetric, src *rand.Rand) (*Normal, bo
 // ConditionNormal returns {nil, false} if there is a failure during the update.
 // Mathematically this is impossible, but can occur with finite precision arithmetic.
 func (n *Normal) ConditionNormal(observed []int, values []float64, src *rand.Rand) (*Normal, bool) {
+	if len(observed) == 0 {
+		panic("normal: no observed values")
```

Comment:
The way I typically speak, zero is plural if the number is normally plural, and singular if the number is usually singular. Here, since it accepts a slice, I pluralize it.

That said, you're right and it's fixed.